Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM node:8-onbuild
+
+EXPOSE 8080
+
+CMD [ "npm", "start" ]
```

Comment:
Would this run `npm start` twice? Looks like the [8-onbuild][1] already runs `npm start`.

[1]: https://github.com/nodejs/docker-node/blob/39a5c8a3be7fff2ddc67a2e72919d0a3841b235f/8.9/onbuild/Dockerfile

Reply:
Good catch, this would override the base images `CMD` instruction. I'll remove this one